lib/common/http/webserver.ts

```diff
@@ -80,6 +80,7 @@ export const webserver = (config, altPort: number) => {
         },
         app,
       );
+  server.maxHeadersCount = 0; // fix https://security.snyk.io/vuln/SNYK-JS-WS-7266574
```

Comment:
q: do we have an upgrade path?

> Upgrade ws to version 5.2.4, 6.2.3, 7.5.10, 8.17.1 or higher.

Reply:
we do but it requires primus and other libraries update which bring a lot of breaking changes, which we can't accomodate at this time.